tests/integration_tests/util.py

```diff
@@ -8,12 +8,14 @@
 from functools import lru_cache
 from itertools import chain
 from pathlib import Path
-from typing import Set
+from typing import TYPE_CHECKING, Set
 
 import pytest
 
 from cloudinit.subp import subp
-from tests.integration_tests.instances import IntegrationInstance
+
+if TYPE_CHECKING:
```

Comment:
Why this? Performance optimization? Can we add a comment or something to the commit message? Also if this isn't required for enabling coverage maybe it should go in its own commit?

Reply:
Avoid circular import. I almost made it a separate commit...so...I guess I will :smile:

Reply:
Could we please include a comment that it's for avoiding circular import?